site/content/resources/security-threat-model.md

```diff
@@ -95,7 +95,7 @@ We maintain a CI pipeline that runs golangci-lint including the usual set of Go
 ## Conclusion
 
 The Contour team works hard to understand the project's security context and keep up with the state of the art for Kubernetes security.
-The team hopes that this examination of our security model provides some insight into both how we develop Contour and what adminsitrators should be thinking about to run Contour in as secure way as possible.
+The team hopes that this examination of our security model provides some insight into both how we develop Contour and what administrators should be thinking about to run Contour in as secure way as possible.
```

Comment:
This one got fixed in https://github.com/projectcontour/contour/pull/4382 (just merged), but I think GitHub will handle it fine.